src/data-seeding/certificates/source/test/UsingSignatureHandlebars.svg

```diff
@@ -0,0 +1,57 @@
QMDAF0FAQAAcksAAgJwAkwMCwEADw0LEgwSCQgHBQAKAQoHDBQrEzIWFRQGIyMVIxEXIxUzMjY1NM5dWmJhQTmARzlFTAIUT0tPV9QCFDDgMUJtAAACAA0AAAFwAhQADQAVADhANQEBAgQBSgcBBAACAQQCZQAFBQBdAAAAcksGAwIBAXABTA8OAAASEA4VDxUADQANEREmCAwXKzM3JiY1NDYzMxEjNSMHEzM1IyIGFRQNiy05YVmEOGmBm09IPkbtDUU8TE397OHhARDUMDhsAAAAAAIADgAAAXACFAANABUAO0A4BgEFAgFKAAIABQQCBWUDAQEBcksHAQQEAF0GAQAAcABMDw4BABIQDhUPFQwLCgkIBwANAQ0IDBQrMyImNTQ2NyczFzM1MxEnMzUjIhUUFuhYXjosikCBaTiETE5+QktLPEkL7uHh/eww1Gw3MQAAAQAQAAABggIUAAcAG0AYAwEBAQJdAAICcksAAABwAEwREREQBAwYKzMjESM1IRUj5jmdAXKcAeQwMAAAAQBC//kBxgIUABEAIUAeBAMCAQFySwACAgBfAAAAeABMAAAAEQARIxMjBQwXKwERFAYjIiY1ETMRFBYzMjY1EQHGZGBdYzpGREREAhT+qFppaF0BVv6qSUxORQFYAAD//wAmAFYCRAHoAYcAUQAmAjoAAMAAQAAAAAAJsQABuAI6sDMrAAAA//8AGwAwAvUBwwGHAL4C6//kAABAAMAAAAAACbEAA7j/5LAzKwAAAP//ADT/zQJSAnkBhwBQADQCywAAwABAAAAAAAmxAAG4AsuwMysAAAD//wAAAAABuAIUAEYAWQAAP0RAAAABAAv//wJ2AhQAIQAnQCQcEAYDAwABSgIBAgAAcksFBAIDA3ADTAAAACEAIREZGREGDBgrFwMzExYWFzM2NjcTMxMWFhczNjY3EzMDIwMmJicjBgYHA42CN00KDwQCBBEJXDpYChADAwMPCko2fjpcCREEAgQQCGABAhX+uShHGhhIIgFO/rAkQBoXRygBSP3rAVoiPxUVQSH+p///ACUAAAGOAhQCBgBdAAAAAQAY//UBeAIUABoAQkA/AQEEBQwBAgMLAQECA0oXAQABSQAAAAMCAANnAAQEBV0GAQUFcksAAgIBXwABAXgBTAAAABoAGhIkJSQSBwwZKwEVBxYWFRQGIyImJzUWFjMyNjU0JiMjNTcjNQFosFhobVYyURogTi5ATFdJMqzyAhQquARFUVFSFQ42EBk7Oj8qLbYuAAEAI//4AWQCHAAmADtAOBEBAQIjGxoQBwYGAwEkAQADA0oAAQECXwACAnpLAAMDAF8EAQAAeABMAQAhHxUTDgwAJgEmBQwUKxciJjU0Njc1NjY1NCYjIgYHJzY2MzIWFRQGBxUGBhUUMzI2NxUGBtNSXk8+OEA2MSA/GhAcSSdJVUk5QkN3KEogGEoIRUQ+PxIkDiovKSgSDTANEkQ7Oz4UKA0sLVsVDTQLEgABABL/9gGzAh4AKQArQCgcFg4GBAAEFQcCAQACSgAEBHpLAwEAAAFfAgEBAXgBTCglJiUiBQwZKyUWFjMyNjcVBgYjIiYmJw4CIyImJzUWFjMyNjcmJjU0NjYzMhYWFRQGAQkpORUQGQoIHhEWKzQjIjUtFhMcCQgaEBo4KDQ6IkIwMEEhPpQ8MQgFMgQIFDQwMDQUCAUxBAkxOz51OCxJKytJLDh4AAABAEgAAAFoAhQABQAfQBwAAAACXQMBAgJySwABAXABTAAAAAUABRERBAwWKwEVIxEjEQFo5zkCFDH+HQIUAAAAAQACAAABsgIWAAwAIUAeCAEBAAFKAAAAcksDAgIBAXABTAAAAAwADBERBAwWKzMTMxMjAyYmJwYGBwMCwDS8PIQGDQMEDQeHAhb96gGAECYUFCYS/oIAAAEASAAAAbwCFAAHABtAGAABAQNdAAMDcksCAQAAcABMEREREAQMGCshIxEhESMRIQG8N/78OQF0AeT+HAIUAP//AEgAAAGFAhQCBgUEAAAAAQAuAAACCQIUABcAK0AoBgEEAgEAAQQAZwgHBQMDA3JLAAEBcAFMAAAAFwAXERETExEREwkMGysBFRQGIxUjNSImNTUzFRQWMxEzETI2NTUCCWVuNnFhN0tQNk5MAhSwWmiiomhZsa9NRgFC/r5GTa8AAAABAAL/+AGqAhQAGAB0QAoOAQMBDQEAAwJKS7APUFhAGgABAQRdAAQEcksAAABwSwADAwJfAAICeAJMG0uwHVBYQBYAAQEEXQAEBHJLAAMDAF8CAQAAcABMG0AaAAEBBF0ABARySwAAAHBLAAMDAl8AAgJ4AkxZWbcWJCYREAUMGSshIxEjDgIHBgYjIiYnNRYzMjY3PgI3IQGqOKMFDQ8KEDIzDBgJFBUcGw4IEQ8HAQsB5C5wdDVYTQUEMQs7UC52gD7//wAAAR8BdQLNAUcAJAAAAR8pmiZmAAmxAAK4AR+wMysAAAD/////AR8B4gLLAUcAiAAAAR8pmiZmAAmxAAK4AR+wMysAAAD//wA9AR8BYgLLAUcAJQAAAR8pmiZmAAmxAAO4AR+wMysAAAAAAwADAR8BgwLLABMAHAAjAGZLsCpQWEAiBgICAAkKBQMDCAADZQAHBwFdAAEBl0sACAgEXQAEBJsETBtAIAABAAcAAQdlBgICAAkKBQMDCAADZQAICARdAAQEmwRMWUAWAAAjIR8dHBoWFAATABMkERQhEQsOGSsTNTM1MzIWFRQHMxUjFhUUBiMjNTczMjY1NCYjIxEzMjU0JyMDOnlRTjBeUTBVRYsmXDwzOT5UYnRpbQHwHr0xNzsaHhk+PzvRHikoKiT+kF5QBQD//wA9AR8BjgLLAUcAJwAAAR8pmiZmAAmxAAK4AR+wMysAAAD//wA9AR8BLgLLAUcAKAAAAR8pmiZmAAmxAAG4AR+wMysAAAD//wAlAR8BFQLLAUcDcAAAAR8pmiZmAAmxA
```

Comment:
Can we remove everything other than the signatures from this file?

Reply:
![image](https://github.com/opencrvs/opencrvs-farajaland/assets/32668488/d7dc7dc1-f93a-4a87-a8e7-97b22a01bf21)

The svg actually doesn't work without this. So I guess, we might actually need this.

Reply:
Actually, it's okay to remove the style tag from the SVG. It works and it just shows a bolder font.